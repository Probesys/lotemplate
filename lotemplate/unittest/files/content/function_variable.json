{
  "test(\"jean\")": {
    "type": "text",
    "value": "test jean value"
  },
  "gloubi(test)": {
		"type": "text",
		"value": "gloubi value"
  },
  "user(odoo.test.glou)": {
		"type": "text",
		"value": "user_value"
  },
  "user($foo\\(bar\\))": {
		"type": "text",
		"value": "extended value"
  },
  "simple_var": {
		"type": "text",
		"value": "valeur simple"
  }
}