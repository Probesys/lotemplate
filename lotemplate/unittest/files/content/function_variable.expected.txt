test jean value

gloubi value

user_value

on va tester plusieurs variables sur la même ligne :  extended value et user_value et une variable simple : valeur simple
